### PR TITLE
Editor delete file user flow

### LINKED ISSUE #352: Editor delete script/transaction/contract user flow
Is your feature request related to a problem? Please describe.
v2Playground editor requires functionality to delete files

Describe the solution you'd like

- add CTA to right click action to delete file from playground editor file explorer
- implement api request and reflect deleted file in view.
- has api dependency

### PR DESCRIPTION
**Pull request by MrDSGC**
Closes: #352

## Description

- implement file explorer submenu
- implement editor delete file user flow
- implement editor edit filename user flow
______
image

image

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: src/components/Editor/FileExplorer/MenuList.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { SXStyles } from 'src/types';
 import { Box, Flex } from 'theme-ui';
 import { getParams } from 'util/url';
 import useKeyPress from '../../../hooks/useKeyPress';
-import FileExplorerSubMenu from './FileExplorerSubMenu';
+import { ContextMenu } from '../../ContextMenu';
```
Comment (bthaile) [inline src/components/Editor/FileExplorer/MenuList.tsx:24]: ❤️  using context menu component

**Review by devbugging — COMMENTED** [review]
Good work

File: src/components/ContextMenu.tsx [hunk 2]
```diff
@@ -52,14 +56,23 @@ const styles: SXStyles = {
   },
 };
 
-export const ContextMenu = ({ options, showDotDotDot }: ContextMenuType) => {
+export const ContextMenu = ({ options, showEllipsis }: ContextMenuType) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
 
   const clickOption = (onClick: Function, args: any[] = []) => {
+    console.log(args)
+    console.log(onClick)
```
Comment (gpavlov2016) [inline src/components/ContextMenu.tsx]: Are the console logs here to stay or just temporary for debugging?

Comment (MrDSGC) [reply src/components/ContextMenu.tsx]: nice catch. was a dev artifact

File: src/components/ContextMenu.tsx [hunk 3]
```diff
@@ -52,14 +56,23 @@ const styles: SXStyles = {
   },
 };
 
-export const ContextMenu = ({ options, showDotDotDot }: ContextMenuType) => {
+export const ContextMenu = ({ options, showEllipsis }: ContextMenuType) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
 
   const clickOption = (onClick: Function, args: any[] = []) => {
+    console.log(args);
+    console.log(onClick);
```
Comment (bthaile) [inline src/components/ContextMenu.tsx]: prob can remove console logs

File: src/components/ContextMenu.tsx [hunk 4]
```diff
@@ -9,29 +9,33 @@ type ContextMenuOptionsType = {
...
+  menuAbsolute: {
     display: 'flex',
     flexDirection: 'column',
     borderRadius: '8px',
     border: `1px solid ${theme.colors.borderColor}`,
     boxShadow: `0px 4px 40px rgba(0, 0, 0, 0.08)`,
     position: 'absolute',
-    zIndex: '11',
-    left: '150px',
+    zIndex: '100',
```
Comment (bthaile) [inline src/components/ContextMenu.tsx]: no todos, just a reminder 
we need to make zIndex a constant in theme object so we can use constants. I didn't originally cuz I wanted to see what all layer we need.

**Review by bthaile — APPROVED** [review]
changes look good, one comment to remove console logs if want

**Pull request merged**